src/plugins/controls/common/options_list/types.ts

```diff
@@ -15,9 +15,11 @@ export const OPTIONS_LIST_CONTROL = 'optionsListControl';
 
 export interface OptionsListEmbeddableInput extends DataControlInput {
   selectedOptions?: string[];
+  existsSelected?: boolean;
```

Comment:
Note that, rather than storing `existsSelected` as a separate property, I **could** have included this status in the `selectedOptions` array. However, after further investigation, there are a few reasons why I opted to keep this logic separate:
1.